### ISSUE TITLE
Avatar - Image Alt text Accessibility

### ISSUE THREAD

**Issue by harrisonwillettbetterworks** [primefaces/primevue#3593]
### Describe the bug
The Avatar component has a WCAG SC 1.1.1 failure found here:
https://github.com/primefaces/primevue/blob/62672251e86048905481e42484cd1f8d48270c05/components/avatar/Avatar.vue#L6

When running Axe Dev Tools on an Avatar component with an image a failure of SC 1.3.1 is shown. There are a few solutions to this including adding alt text `:aria-label="ariaLabel"` or adding a role like  `role="none"` or `role="presentation"`. I can see potential issues with either option depending on the implementation in an application. You may want to provide the option to choose which is used depending on design pattern.
 
 ### Proposed Fix
 
-  `<img v-else-if="image" :alt="ariaLabel" :src="image" @error="onError" />`
-  `<img v-else-if="image" role="presentation" :src="image" @error="onError" />`

### PrimeVue version
3.22.4

### Vue version
3.x

### Language
TypeScript
